x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -320,6 +320,8 @@ const allowlistEventFields: AllowlistFields = {
     Ext: {
       code_signature: true,
       malware_classification: true,
+      quarantine_result: true,
```

Comment:
We also need `malware_signature` in here, I believe.

Reply:
This will be updated in a separate PR :)